CutestTorture/Assets/Scripts/Player/AxisAlignedMovement.cs

```diff
@@ -0,0 +1,81 @@
+using UnityEngine;
+
+namespace Player
+{
+    public class AxisAlignedMovement : MonoBehaviour
+    {
+        public float maxVelocity;
```

Comment:
You have an incorrect branching scheme. The in-class logic (even though you made it yourself) you have been located in the `master` branch